### ISSUE TITLE
Seems Like a Virus

### ISSUE THREAD

**Issue by arunseshan** [Klaudit/mactype#136]
Downloaded and installed on a Windows 10 machine and it crashed. Had to do a complete System Restore. Thanks man!!!

**Comment by Erinaaaaaaa** [comment]
Nope, not a virus. This version isn't just compatible with the Anniversary Update, also known as the Windows 10, version 1607 update, which is most likely the one you're using (the linked repo has an issue for it). also, look at [snowie2000's repo](https://github.com/snowie2000/mactype/) for the latest mactype updates. Just a little warning though: you'll have to build the dll yourself.

**Comment by Erinaaaaaaa** [comment]
Like um, Windows 10 didn't even exist at the time of the last release in this repo, so yeah
